sys/kern/kern_cheri_revoke.c

```diff
@@ -579,7 +579,9 @@ kern_cheri_revoke(struct thread *td, int flags,
 	     ((myst == CHERI_REVOKE_ST_SS_LAST) ||
 	      (myst == CHERI_REVOKE_ST_LS_CLOSING))) {
 
-		// XXX Assert all capdirty PTEs have LCLG equal to GCLG
+#ifdef INVARIANTS
```

Comment:
Hm, normally that'd be DIAGNOSTIC instead

Reply:
Indeed, will make that switch.